rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -28,24 +28,59 @@ Inherent to this feature is a concept of how scoping and naming is handled for n
 This RFC introduces two concepts:
 
 - Element usage from other files, and
-- Libraries
+- Packages
 
 ## Motivation
 
 Right now, Jayvee users can only pack their whole Jayvee model into one file.
 The challenge is two-fold:
 
 1. Larger projects will become unmaintainable quite quickly, as the elements cannot be organized into multiple files.
-2. Without distribution to multiple files, there is no possibility to reuse models of other projects.
-
-For example, users will be enable to build libraries of valuetypes that can be reused across multiple projects instead of copying the code.
+2. Without distribution to multiple files, there is no possibility to later reuse models of other projects.
 
 ## Explanation
 
+### Element visibility
+
+We distinguish different kinds of visibilities of elements:
+
+- `file-private`: usable only within the same file
+- `file-published`: usable also in other files of same project
+- `package-private`: usable only within the package
+- `package-published`: usable also in other locations (since `package` is always `file-published`)
+
+By introducing the concept of `packages`, most elements can be defined on three levels in a Jayvee file:
+
+1. On the root level of the file
+2. Within a pipeline
+3. (new) Within a package
+
+The name of an element is given by its definition.
+The **qualified name** is constructed by prepending container structures in this pattern: `<container name>.<element name>`, e.g., `MyDomainPackage.MyDomainSpecificValuetype`.
+
+**Access paths:**
+
+- root level elements can access
+  - root level elements by their name
+  - and elements of packages by their qualified name
+- elements within a package can access
+  - root level elements by their name
+  - and elements of other packages by their qualified name
```

Comment:
Good catch. Refactore the section with green checkmarks and red crosses for better readability in one text section rather than 2 (allowed/unallowed access paths)